test/test_extract.py

```diff
@@ -109,8 +108,6 @@ class TestExtractFileRpm(TestExtractorBase):
     """ Tests for the rpm file extractor """
 
     def setUp(self):
-        self.assertTrue(inpath("rpm2cpio"), msg="Required tool 'rpm2cpio' not found")
```

Comment:
We probably still want these on linux and should maybe just if windows / else on these?

Reply:
This change could probably be delayed to another pr, though.